src/develop/develop.c

```diff
@@ -1122,8 +1158,12 @@ static void _dev_add_history_item(dt_develop_t *dev,
 {
   if(!darktable.gui || darktable.gui->reset) return;
 
+  gchar *saved_name = g_strdup(module->multi_name);
```

Comment:
@TurboGit this needs to be saved _before_ it gets potentially changed by `_dev_auto_module_label`. Now the strcmp below always returns 0, so `target` is always passed to `_dev_undo_start_record_target` causing updates of the history list  in the left panel to be skipped on multiple changes of the same widget and the multi-name shown there to get out of sync with the name on the module itself.

Reply:
Arrrrgggg :( Code refactoring with side-effects is not easy and I completely missed this. Will fix, and of course thanks for catching.

Reply:
Now fixed, do not hesitate to double check :)